[windows] ABI mismatch between compiled bindings & skia (`clang::trivial_abi`)

On 5f17a054e21ff0cd5b90146c218a13e3a7f3825d, when compiling for windows with `SKIA_DEBUG=1`, a simple application using the `wrap_backend_render_target` function will freeze. The callstack looks like this at the Rust/C/C++ interface
```
...
kyute_async.exe!SkSurfaces::WrapBackendRenderTarget(GrRecordingContext * rContext, const GrBackendRenderTarget & rt, GrSurfaceOrigin origin, SkColorType colorType, sk_sp<SkColorSpace> colorSpace, const SkSurfaceProps * props, void(*)(void *) relProc, void * releaseContext) Line 733	C++
kyute_async.exe!C_SkSurfaces_WrapBackendRenderTarget(GrRecordingContext * context, const GrBackendRenderTarget * backendRenderTarget, GrSurfaceOrigin origin, SkColorType colorType, SkColorSpace * colorSpace, const SkSurfaceProps * surfaceProps) Line 413	C++
kyute_async.exe!skia_safe::gpu::ganesh::surface_ganesh::wrap_backend_render_target<skia_safe::prelude::RCHandle<skia_bindings::SkColorSpace>>(skia_safe::prelude::RCHandle<skia_bindings::GrRecordingContext> * context, skia_safe::prelude::Handle<skia_bindings::GrBackendRenderTarget> * backend_render_target, skia_bindings::GrSurfaceOrigin origin, skia_safe::core::color_type::ColorType color_type, skia_safe::prelude::RCHandle<skia_bindings::SkColorSpace> color_space, enum2$<core::option::Option<ref$<skia_safe::core::surface_props::SurfaceProps>>> surface_props) Line 125	Rust
...
```

When debugging it seems that the `SkColorSpace` object gets corrupted when passed to `SkSurfaces::WrapBackendRenderTarget`, I suspect that this is due to the presence of `[[clang::trivial_abi]]` on sk_sp which changes the calling convention for parameters of this type. Skia itself is compiled with clang, but the bindings in `skia-bindings/src` are compiled with MSVC which does not recognize (and ignores) this attribute, resulting in the mismatch:
```
.cargo\git\checkouts\rust-skia-35c6c5f9d522ea9f\5f17a05\skia-bindings\skia\include/core/SkRefCnt.h(220): warning C5030: attribute [[clang::trivial_abi]] is not recognized
```

Related issue on skia-discuss: https://groups.google.com/g/skia-discuss/c/LGiqnXLzjXA . I think the easiest way forward would be to build skia with `is_trivial_abi = false`.

--- a/skia-bindings/build_support/platform/windows.rs
+++ b/skia-bindings/build_support/platform/windows.rs
@@ -44,6 +44,11 @@ impl PlatformDetails for Msvc {
             }
         }
 
+        // Disable `[[clang::trivial_abi]]` because it leads to ABI mismatches if the
+        // bindings are compiled with a compiler other than clang (e.g. MSVC).
+        // (see <https://groups.google.com/g/skia-discuss/c/3rpeWuPcD9Y/m/CySLakaTAAAJ>)
+        builder.arg("is_trivial_abi", no());
+
         // Code on MSVC needs to be compiled differently (e.g. with /MT or /MD)
         // depending on the runtime being linked. (See
         // <https://doc.rust-lang.org/reference/linkage.html#static-and-dynamic-c-runtimes>)

